### DISCUSSION TITLE
Weird empty space between dock and apps

### DISCUSSION THREAD

**Discussion by NikunjK8** [LawnchairLauncher/lawnchair#4234] [Q&A] [answered]
Is this actually a bug or I'm doing something wrong? Need help

![IMG_20240405_224343](https://github.com/LawnchairLauncher/lawnchair/assets/99175083/b594da71-da42-4c9b-8e52-3ed1f59f2b4b)

**Comment by Matth7878** [comment]
Same thing here.
Maybe space is there because it's where the search bar would be. When you don't add a search bar you end up with an empty space.

**Reply by NikunjK8** [reply]
For me it randomly appears. Sometimes the layout is perfectly fine, other times it becomes this

**Comment by lineinthesand** [comment]
This is a link to the relevant bug report, in case anyone wonders whether there's already one been submitted:
https://github.com/LawnchairLauncher/lawnchair/issues/4836

**Comment by tgex0** [comment, marked as answer]
Try currently nightly. Fixed in https://github.com/LawnchairLauncher/lawnchair/commit/38934f824fec6da9cabc75f118449c94ae9afd1b.
Reduce 'Page indicator height' in Dock settings.

![Screenshot_Lawnchair](https://github.com/user-attachments/assets/bfeb4864-d9dc-4a6f-944f-759906c720d5)

**Reply by NikunjK8** [reply]
This occurs and go away randomly so not really possible to reproduce but I'll let you know if it happens again.
